test_runner/performance/test_perf_pgbench.py

```diff
@@ -70,31 +69,75 @@ def run_pgbench(env: PgCompare, prefix: str, cmdline):
 # the test database.
 #
 # Currently, the # of connections is hardcoded at 4
-def run_test_pgbench(env: PgCompare, scale: int, duration: int):
-
-    # Record the scale and initialize
+def run_test_pgbench(env: PgCompare, scale: int, duration: int, filter: str = "all"):
```

Comment:
`"all"` doesn't run all tests, which is maybe why `test_pgbench` and `test_pgbench_flamegraph` are failing in CI